src/components/PasswordInput.tsx

```
@@ -138,11 +138,28 @@ const PasswordInput = ({ value, onRefresh }: PasswordInputProps) => {
 
-    const handleRefreshPassword = (length: number = PASSWORD_LENGTH) => {
-        const { password, hasRepeatingChars } =
-            generatePassword(PASSWORD_LENGTH);
-        onRefresh(password);
-        setHasRepeatingChars(hasRepeatingChars);
+   const handlePasswordCopy = () => {
+        try {
+            if (value.length === 0) toast.error("비밀번호가 비어있습니다.");
+            else {
+                navigator.clipboard.writeText(value);
+                toast.success("비밀번호가 복사되었습니다.");
```

Biome (1.9.4)

[error] Shouldn't redeclare 'handlePasswordCopy'. Consider to delete it or rename it.

'handlePasswordCopy' is defined here:


(lint/suspicious/noRedeclare)